### PR TITLE
New versions of Temporal and Indicators

### PR DESCRIPTION
**Pull request by dysonance**
Still having to do this from the command line (PkgDev.publish still not working for me), so if any issues just let me know and I can jump on it.

### CODE REVIEW

**Comment by tkelman** [comment]
It's discouraged to extend Base methods like `diff` on signatures where all the types are also from Base, as that can change the behavior of unrelated code depending on whether or not your package has been imported.

**Comment by dysonance** [comment]
@tkelman Yeah I can see that. I will change it back. Should I go through the whole process of retagging and resubmitting the PR? Or is there a more efficient way of making that change?

**Comment by tkelman** [comment]
You could either make it an 0.1.2 tag and add it to this branch with a new commit, or modify the existing 0.1.1 tag since it hasn't been published yet

**Comment by dysonance** [comment]
@tkelman Did that do the trick?

**Comment by tkelman** [comment]
yep, looks good

File: Temporal/versions/0.1.2/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5-
```
Comment (tkelman) [inline Temporal/versions/0.1.2/requires]: ah but as soon as you don't need to support prereleases any more, would be good to drop the trailing dash here from REQUIRE - no rush at all though

**Comment by nalimilan** [comment]
I've discovered these very interesting packages -- I was intrigued by the name Indicators.jl in the PR's title. I'd just like to point out that it would likely be easier to find if it's name was more explicit, like TimeSeriesIndicators.jl or FinancialIndicators.jl.

**Pull request merged**